hq/app/logic/IamRemediation.scala

```diff
@@ -40,9 +41,24 @@ object IamRemediation extends Logging {
     * Looks through the credentials report to identify users with Access Keys that are older than we allow.
     */
   def identifyUsersWithOutdatedCredentials(awsAccount: AwsAccount, credentialReportDisplay: CredentialReportDisplay, now: DateTime): List[IAMUser] = {
-    ???
+    credentialReportDisplay.machineUsers.filter(user => hasOutdatedMachineKey(List(user.key1, user.key2), now)).toList ++
```

Comment:
Do we need to `toList` at the end, here?

Looks great though.

Reply:
Yes, otherwise it outputs a `Seq`.